frontend/dashboard/utils/repoUtils/repoUtils.ts

```diff
@@ -20,23 +21,33 @@ export const getReposLabel = ({
   orgs,
   t,
   isDatamodelsRepo = false,
+  isResourcesRepo = false,
 }: GetReposLabel) => {
+  const orgName =
+    orgs.length > 0 && orgs.find((org) => org.username === selectedContext)?.full_name;
+  const stringsConfig = {
+    all: t('dashboard.all_apps'),
+    mine: t('dashboard.my_apps'),
+    org: orgName ? t('dashboard.org_apps', { orgName }) : t('dashboard.apps'),
+    ...(isDatamodelsRepo && {
+      all: t('dashboard.all_datamodels'),
+      mine: t('dashboard.my_datamodels'),
+      org: orgName ? t('dashboard.org_datamodels', { orgName }) : t('dashboard.datamodels'),
+    }),
+    ...(isResourcesRepo && {
+      all: t('dashboard.all_resources'),
+      mine: t('dashboard.my_resources'),
+      org: orgName ? t('dashboard.org_resources', { orgName }) : t('dashboard.resources'),
+    }),
+  };
```

Comment:
Nice refac! I loved that you refactored this to use hash-map lookup. It was a little harder to read the properties when they were combined directly. What do you think of splitting each translation into its object? Then we can create our translation object which is concatenated. Please take a look at my example below.

Then we could create two new types which ensure type safety. I have also changed the self/mine to be self. Then we are using the same convention/word as before. Then we do not have two words that are the same thing.

I did also rename the stringsConfig to be named *TranslationMap, to make it clear that this is all about translation/texts. 😊 

The reason for this suggestion is just all about readability and may be easier to maintain over time. What do you think?

```suggestion
type TranslationMapKey = SelectedContextType & { org: string }
type TranslationMap = Record<TranslationMapKey, string>
const dataModelTranslationMap: TranslationMap = {
    all: 'Alle datamodeller',
    mine: 'Mine datamodeller',
    org: orgName ? 'Datamodeller for Org' : 'Datamodeller',
};

const resourcesTranslationMap: TranslationMap = {
    all: 'Alle ressurser',
    mine: 'Mine ressurser',
    org: orgName ? 'Resurrser for Org' : 'Ressurser',
};

const appsTranslationMap: TranslationMap = {
    all: 'Alle apper',
    mine: 'Mine apper',
    org: orgName ? 'Apper for Org' : 'Apper',
};

const concatenatedTranslationMap: TranslationMap = {
    appsTranslationMap,
    ...(isResource && resourcesTranslationMap || {}),
    ...(isDatamodels && dataModelTranslationMap || {})
}

 if (selectedContext === SelectedContextType.All) {
    return concatenatedTranslationMap.all;
  }
```

Reply:
agree, this should make it easier to read!

Reply:
@WilliamThorenfeldt yes for sure, I was just to lacy to add them within my example. 😓